packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -383,107 +383,10 @@ class _Elements {
-    //   // final Paragraph fill = createParagraph(
-    //   //   value,
-    //   //   lastTextInfo.style,
-    //   //   lastTextInfo.style.fill,
-    //   // );
-    //   // final Paragraph stroke = createParagraph(
-    //   //   value,
-    //   //   lastTextInfo.style,
-    //   //   DrawablePaint.isEmpty(lastTextInfo.style.stroke)
-    //   //       ? transparentStroke
-    //   //       : lastTextInfo.style.stroke,
-    //   // );
-    //   // parserState.currentGroup!.children!.add(
-    //   //   DrawableText(
-    //   //     parserState.attribute('id', def: ''),
-    //   //     fill,
-    //   //     stroke,
-    //   //     lastTextInfo.offset,
-    //   //     lastTextInfo.style.textStyle!.anchor ??
-    //   //         DrawableTextAnchorPosition.start,
-    //   //     transform: lastTextInfo.transform?.storage,
-    //   //   ),
-    //   // );
-    //   // lastTextWidth = fill.maxIntrinsicWidth;
-    // }
-
-    // void _processStartElement(XmlStartElementEvent event) {
-    //   _TextInfo? lastTextInfo;
-    //   if (textInfos.isNotEmpty) {
-    //     lastTextInfo = textInfos.last;
-    //   }
-    //   final Point currentPoint = _parseCurrentPoint(
-    //     parserState,
-    //     lastTextInfo?.offset.translate(lastTextWidth, 0),
-    //   );
-    //   AffineMatrix? transform =
-    //       parseTransform(parserState.attribute('transform'));
-    //   if (lastTextInfo?.transform != null) {
-    //     if (transform == null) {
-    //       transform = lastTextInfo!.transform;
-    //     } else {
-    //       transform = lastTextInfo!.transform!.multiplied(transform);
-    //     }
-    //   }
-
-    //   final DrawableStyle? parentStyle =
-    //       lastTextInfo?.style ?? parserState.currentGroup!.style;
-
-    //   textInfos.add(_TextInfo(
-    //     parserState.parseStyle(
-    //       parserState.rootBounds,
-    //       parentStyle,
-    //     ),
-    //     currentPoint,
-    //     transform,
-    //   ));
-    //   if (event.isSelfClosing) {
-    //     textInfos.removeLast();
-    //   }
-    // }
-
-    // _processStartElement(parserState._currentStartElement!);
-
-    // for (XmlEvent event in parserState._readSubtree()) {
-    //   if (event is XmlCDATAEvent) {
-    //     _processText(event.text.trim());
-    //   } else if (event is XmlTextEvent) {
-    //     final String? space =
-    //         getAttribute(parserState.attributes, 'space');
-    //     if (space != 'preserve') {
-    //       _processText(event.text.trim());
-    //     } else {
-    //       _processText(event.text.replaceAll(_trimPattern, ''));
-    //     }
-    //   }
-    //   if (event is XmlStartElementEvent) {
-    //     _processStartElement(event);
-    //   } else if (event is XmlEndElementEvent) {
-    //     textInfos.removeLast();
-    //   }
-    // }
+    assert(parserState.currentGroup != null);
+    if (parserState._currentStartElement!.isSelfClosing) {
+      return;
```

Comment:
Some guidance may be required 😄

Reply:
getting there

Reply:
I'll need to do something more similar to the left to correctly handle nested spans. I think it could be more or less handled with the attribute map though